### PR TITLE
WIP: pelux.xml: bump meta-pelux

### PR DESCRIPTION
**Pull request by thenor**
Adds the following commits:
* packagegroup: Change to use debug-utils for dev images

Signed-off-by: Therese Nordqvist <therese.nordqvist@pelagicore.com>

### CODE REVIEW

**Comment by thenor** [comment]
Awaiting the bigger bump in master before starting the review.

File: pelux.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
   <project remote="github"
            revision="28f52b1451c6bebe46224b81dc6849babf286ce1"
-           upstream="pyro"
+           upstream="master"
```
Comment (tobsan) [inline pelux.xml]: Why this change when bumping meta-pelux?

Comment (thenor) [reply pelux.xml]: I didn't realize until after I had made the PR that another PR bumped the entire internal layers, and also updated the "upstream" branch name. It is a typo on master.

**Comment by thenor** [comment]
I accidentally closed this, but I will need to rebase when the bump to rocko is done, so I will make a new PR when that is done.
